src/ops/fuelup_self.rs

```diff
@@ -18,6 +23,68 @@ pub fn attempt_install_self(download_cfg: DownloadCfg, dst: &Path) -> Result<()>
     Ok(())
 }
 
+/// Removes the fuelup directory from $PATH
+fn remove_fuelup_from_path() -> Result<()> {
+    for shell in Shell::all() {
+        for rc in shell.rc_files().into_iter().filter(|c| c.is_file()) {
+            let file = read_file("rcfile", &rc)?;
+            let mut is_modified = false;
+            let new_content = file
+                .lines()
+                .filter(|line| {
+                    if line.contains("PATH") && line.contains(FUELUP_DIR) {
+                        is_modified = true;
+                        false
+                    } else {
+                        true
+                    }
+                })
+                .collect::<Vec<_>>()
+                .join("\n");
+            if is_modified {
+                println_warn(format!(
+                    "{} has been updated to remove fuelup from $PATH",
+                    rc.display()
+                ));
+                write_file(rc, &new_content)?;
+            }
+        }
+    }
+    Ok(())
+}
+
+pub fn self_uninstall(force: bool) -> Result<()> {
+    println!(
+        r#"Thanks for hacking in Sway!
+This will uninstall all Sway toolchains and data, and remove, {}/bin from your PATH environment variable."#,
+        canonical_fuelup_dir()?,
+    );
+    if force || ask_user_yes_no_question("Continue? (y/N)").context("Console I/O")? {
+        let remove = [
+            ("removing fuelup binaries", fuelup_bin_dir()),
+            ("removing fuelup home", fuelup_dir()),
+        ];
+        remove_fuelup_from_path()?;
+
+        for (info, path) in remove.into_iter() {
+            println_warn(info);
```

Comment:
Can we change this to `info` rather than warning so it matches rustup.

Currently: fuelup

```
warning: /Users/joshuabatty/.zshrc has been updated to remove fuelup from $PATH
warning: removing fuelup binaries
warning: removing fuelup home
```

rustup

```
info: removing rustup home
info: removing cargo home
info: removing rustup binaries
info: rustup is uninstalled
```